### ISSUE TITLE
Improve docs on uploading dsym files when the app is built in Xcode

### ISSUE THREAD

**Issue by sindresorhus** [microsoft/appcenter#1224]
I'm using AppCenter for a macOS app, and coming from Crashlytics, I assumed crash symbolication would be handled for me, but apparently this is not the case. The Crashlytics SDK ships with a built-in script that handles this, where you only have to add a one-liner to your "build phase" in Xcode.

There's no mention of symbolication in https://docs.microsoft.com/en-us/appcenter/sdk/crashes/macos I found at the [hard way](https://github.com/microsoft/appcenter/issues/1223) that I had to manually upload a dsym file.

Looking again, I see mention of the AppCenter CLI in https://docs.microsoft.com/en-us/appcenter/diagnostics/ios-symbolication, but it was not clear that `iOS Symbolication` also applied to macOS while just looking at the sidebar.

So what I'm asking is:
- Make it clearer in https://docs.microsoft.com/en-us/appcenter/sdk/crashes/macos that you need to upload dsym files.
- Include a guide (in https://docs.microsoft.com/en-us/appcenter/sdk/crashes/macos) on how to set up AppCenter CLI to upload dsym files as part of the Xcode build process.

A big downside of the AppCenter CLI is that it requires contributors to manually install Node.js and the App Center CLI themselves and this also has to be documented in every project. The Crashlytics SDK script is included in the SDK and *just works* for every contributor.

**Comment by Oddj0b** [comment]
Hi @sindresorhus, thank you for the elaborate description of the doc's suggestion.  One of my colleagues will have a look at it.

**Comment by Fidetro** [comment]
Maybe you should provide an upload dysm script, like firebase

**Comment by antranapp** [comment]
Yes, this is one of the drawback I found out when switching from firebase to appcenter for crash tracking. A script to integrate in Xcode build phase is a very good UX generally.
